### DISCUSSION TITLE
À quelles conditions les logos se colorent-ils?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1464] [Question / réponse] [answered]
Une utilisatrice me demande pourquoi les logos de couleur n'apparaissent pas sur la fiche de sa cantine alors qu'elle a rempli son diagnostic. Je suppose que cela est dû à l'existence de seuils : "si au moins A, B et C sont remplis, alors coloration du logo". Je connais ce seuil pour le logo "Plus de produits de qualité et durable" : au moins 20% de bio et 50% de durable. Mais quel est-il pour les 4 autres volets du diag, @JenniferStephan, @hfroot et @alemangui?

**Comment by hfroot** [comment, marked as answer]
Appro: > 50% durable et qualité, dont 20% bio

gaspi: la cantine a "réalisé un diagnostic sur les causes probables de gaspillage alimentaire" et a au moins une action "de lutte contre le gaspillage alimentaire". En plus, si la cantine sert > 3000 repas par jour, elle "propose une ou des conventions de dons à des associations habilitées d’aide alimentaire"

diversification: 
- si la cantine a au moins un secteur avec la catégorie de 'education' : elle doit fournir au moins un repas vege par semaine
- sinon : elle doit fournir au moins un repas vege par jour

plastic: tous les quatre cas cochés

info: la cantine "informe mes convives sur la part de produits de qualité et durables, entrant dans la composition des repas servis, et sur les démarches d’acquisition de produits issus d'un PAT (projet alimentaire territorial)"

**Reply by ghost** [reply]
Merci pour cette réponse détaillée, @hfroot.

**Reply by valeriemerle** [reply]
Pour les repas végé: **attention pas d'obligation à servir un repas vege par jour à ce jour**. Pour les secteurs hors scolaires (maternelles à lycée mais pas universitaire), pas d'obligation sur un repas vege par semaine. Obligation option vege quotidienne à partir de janvier 2023 pour le secteur "Etat" ses entreprises et etablissements publics (incluant les hôpitaux) et restaurants d'etat.

**Reply by ghost** [reply]
Merci de la précision, @valeriemerle.

**Reply by hfroot** [reply]
@valeriemerle est-ce qu'on doit changer les règles pour le badge diversification alors ?

**Reply by hfroot** [reply]
Discussion reliée : https://github.com/betagouv/ma-cantine/issues/1356
